bouncer/src/repo/manipulator/modelutility/repo_scene_manager.cpp

```diff
@@ -69,16 +74,17 @@ bool SceneManager::commitWebBuffers(
 	}
 
 	std::string errMsg;
-	if (REPO_COLLECTION_STASH_UNITY == geoStashExt)
+
+	if(!resultBuffers.unityAssets.isEmpty())
 	{
-		if (success &= handler->upsertDocument(databaseName, projectName + "." + geoStashExt, resultBuffers.unityAssets,
-			true, errMsg))
+		if (success &= handler->upsertDocument(databaseName, projectName + "." + assetsStashExt, resultBuffers.unityAssets,
```

Comment:
any reason why we set the overwrite flag to false here?

In normal usage we don't ever anticipate to need to overwrite this, but in the event where we want to re-generate the stash for some reason, this would fail without the overwrite flag

Reply:
I have put this back since we have separate asset lists now!